tokenbridge/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM node:16-bullseye-slim
 RUN apt-get update && \
-    apt-get install -y git docker.io
+    apt-get install -y git docker.io python3 chromium build-essential
 WORKDIR /workspace  
-RUN git clone --depth 1 -b v3.1.4 https://github.com/OffchainLabs/arbitrum-sdk.git ./
+RUN git clone -b add-creator-ci https://github.com/OffchainLabs/token-bridge-contracts.git ./
```

Comment:
maybe better to use main or a tagged version, is https://github.com/OffchainLabs/token-bridge-contracts/pull/50 and https://github.com/OffchainLabs/token-bridge-contracts/pull/51/files ready to merge?

Reply:
right will switch to tag. 50 is merged, 51 not yet but that one is not necessary

Reply:
switched to latest tag

Reply:
bumped to 1.1.2